fac29/NisaInvest-TFB-FE
# Nav bar

## Description
I've created a responsive nav bar which displays differently on mobile and desktop. It also has conditional rendering dependent on whether the user is logged in or not. Unfortunately, the code is a bit repetitive at the moment but I didn't want to spend more time on this than I have already.

Mobile nav:
![image](https://github.com/user-attachments/assets/df7fde68-77d0-48f9-a6b8-09b5a97e94b3)

Desktop nav:
![image](https://github.com/user-attachments/assets/cacdd5b7-cbef-4a6c-b235-08884ae131e5)

Conditional rendering when logged in:
![image](https://github.com/user-attachments/assets/778c829d-b2e0-407b-afae-f33f1101c4a5)
![image](https://github.com/user-attachments/assets/ecb51f4c-83f4-4edc-8613-9c4e16a1d09c)

## Related Issue
Closes #7 

## How Can This Be Tested?
Run `npm install`, `npm run dev` and use browser tools to change the shape of the window to see the responsive nav bar.

To test the conditional rendering, on `NavBar.tsx line 22`, change 'false' to 'true':
```const [isLoggedIn, setLogIn] = useState(false);```
```const [isLoggedIn, setLogIn] = useState(true);```

## Checklist:
